### DISCUSSION TITLE
Question about a cs-app template for new architecture

### DISCUSSION THREAD

**Discussion by k1mmm** [microsoft/react-native-windows#13221] [General]
Hello!

Are there any plans to include a C# template as well in the experimental phase of the new architecture? Also, are there any ballpark figures for when you think Milestone 2 will be ready?

Thanks for all the work so far, it's a blast building with React Native for MacOS and Windows!

**Comment by comachrome** [comment]
+1

**Comment by nagibmahfuj** [comment]
+1

**Comment by IamJignesh** [comment]
+1
